File: solutions/go/etl/1/etl.go
package etl

import "strings"

func Transform(in map[int][]string) map[string]int {
	values := map[string]int {}
    for point, letters := range in {
        for _, letter := range letters {
            values[strings.ToLower(letter)] = point
        }
    }
    return values
}

File: solutions/swift/etl/1/Sources/Etl/Etl.swift
class ETL {
  static func transform(_ old: [String: [String]]) -> [String: Int] {
    var new = [String: Int]()
    for (pointsAsStr, letters) in old {
      guard let points = Int(pointsAsStr) else {
        fatalError("Not a valid integer")
      }
      for letter in letters {
        new[letter.lowercased()] = points
      }
    }
    return new
  }
}
